docs/reference/contributing/target/flash.md

```diff
@@ -57,6 +59,10 @@ Bootloader-ready declaration of flash VTOR address:
 #endif
 ```
 
+#### Targets.json Metadata
```

Comment:
- Please put "targets.json" in code format, and make sure it's lower case.
- Please lower case "metadata" for consistent sentence case.